saleor/plugins/manager.py

```diff
@@ -151,7 +151,7 @@ def calculate_checkout_subtotal(
                 line_info.product,
                 line_info.collections,
                 address,
-                checkout.channel,
+                line_info.channel_listing.channel,
```

Comment:
This line can generate additional DB queries. Maby we should store `channel` in `line_info`

Reply:
I would change that in a separate PR - we need to add `channel` to `CheckoutLineInfo` dataclass and then update all other places to use it.

Reply:
Channel is prefetched and it should not generate additional db queries.

Reply:
I think we should change it later anyway - instead of using Django's prefetches, it should be better to load it with data loaders - I think then we will load it only when it is necessary.